test/Microsoft.Azure.Devices.Edge.Test.Common/EdgeModule.cs

```diff
@@ -92,23 +92,31 @@ public Task WaitForStatusAsync(EdgeModuleStatus desired, CancellationToken token
             return WaitForStatusAsync(new[] { this }, desired, token);
         }
 
-        public Task WaitForEventsReceivedAsync(DateTime seekTime, CancellationToken token)
+        public Task<string> WaitForEventsReceivedAsync(DateTime seekTime, CancellationToken token, params string[] requiredProperties)
         {
```

Comment:
you would have to manually pass in null, which shouldn't happen. people rarely pass actual arrays to a params parameter. Besides, this is a test framework, it will just fail the test if you pass in null.